autotest.pm

```diff
@@ -126,6 +126,7 @@ sub loadtest ($script, %args) {
             sys.path.append("$inc")
             sys.path.append("$script_dir")
             import $name
+            print("Using python version: %s" % sys.version)
```

Comment:
IMHO we can use better approaches than an unconditional print. How about using python logging and a log.debug here? I don't see enough value to make that an unconditional print.

Reply:
`log.X` will still mean an unconditional print, python logger by default (and by no other interaction) has the lowest reporting level.

To be able to "skip" that message you'd be willing to do the undoable: alter the global defaults to skip a message print.

Reply:
@okurz if you suggest a different place to put the Python version, I'm cool with that, I honestly don't care about the unconditional print, with an unconditional print, we'd would clearly mark that X module is running with Python (see my answer to @Martchus )

Also this has to be perl.diag, so it shows up in the autoinst-log.txt